src/libraries/System.Private.CoreLib/src/System/Random.cs

```diff
@@ -182,6 +182,113 @@ public virtual void NextBytes(byte[] buffer)
         /// <param name="buffer">The array to be filled with random numbers.</param>
         public virtual void NextBytes(Span<byte> buffer) => _impl.NextBytes(buffer);
 
+        /// <summary>
+        ///   Fills the elements of a specified span with items chosen at random from choices.
+        /// </summary>
+        /// <param name="choices">The items to use to populate the span.</param>
```

Comment:
Ugh I see the name is baked into the API . In en-GB, these would not be choices.